xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -675,8 +675,16 @@ The `GetSingle` method throws an exception if the entire UTF-8 encoded token val
       <Docs>
         <summary>Reads the next JSON token value from the source, unescaped, and transcoded as a string.</summary>
         <returns>The token value parsed to a string, or <see langword="null" /> if <see cref="P:System.Text.Json.Utf8JsonReader.TokenType" /> is <see cref="F:System.Text.Json.JsonTokenType.Null" />.</returns>
-        <remarks>To be added.</remarks>
-        <exception cref="T:System.InvalidOperationException">The JSON token value isn't a string (that is, not a <see cref="F:System.Text.Json.JsonTokenType.String" /> or <see cref="F:System.Text.Json.JsonTokenType.PropertyName" />).
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Returns `null` when <xref:System.Text.Json.Utf8JsonReader.TokenType> is <xref:System.Text.Json.JsonTokenType.Null?displayProperty=nameWithType>.
+
+          ]]></format>
+        </remarks>
+         <exception cref="T:System.InvalidOperationException">The JSON token value isn't a string (that is, not a <see cref="F:System.Text.Json.JsonTokenType.String" />, <see cref="F:System.Text.Json.JsonTokenType.PropertyName" />, or <see cref="F:System.Text.Json.JsonTokenType.Null" />).
```

Comment:
nit: looks like there is misaligned indentation (due to incorrect GitHub suggestion).
```suggestion
        <exception cref="T:System.InvalidOperationException">The JSON token value isn't a string (that is, not a <see cref="F:System.Text.Json.JsonTokenType.String" />, <see cref="F:System.Text.Json.JsonTokenType.PropertyName" />, or <see cref="F:System.Text.Json.JsonTokenType.Null" />).
```